src/pages/projects.jsx

```diff
@@ -15,48 +15,86 @@ const StyledH1 = styled.h1`
     margin-bottom: 20px;
 `;
 
+const StyledP1 = styled.p`

```

Comment:
Rather than `StyledP1` and `StyledP`, can you give these better names so I can know what they're for just by looking at the names? For example, `StyledP1` here is the styling for this page's overview, so a better name could be `PageOverviewStyling`. And `StyledP` is the styling for each project's description, so a better name could be `ProjectDescriptionStyling`.